oak_runtime/src/node/http/tests.rs

```diff
@@ -15,25 +15,37 @@
 //
 
 use super::*;
+use crate::{node::Node, LabelReadStatus::Success};
 use maplit::hashmap;
 use oak_abi::{
-    label::Label,
+    label::{confidentiality_label, tls_endpoint_tag, Label},
     proto::oak::application::{
         node_configuration::ConfigType, ApplicationConfiguration, HttpServerConfiguration,
         NodeConfiguration,
     },
+    Handle,
 };
 use std::fs;
 
 static LOCAL_CA: &str = "../examples/certs/local/ca.pem";
 static GCP_CA: &str = "../examples/certs/gcp/ca.pem";
 
-/// A simple Oak node that responds with 200 (OK) to every request it receives
-struct OakNodeForTest;
+fn get_label(runtime: &RuntimeProxy, handle: Handle) -> Option<Label> {
+    match runtime
+        .get_serialized_channel_label(handle, std::usize::MAX)
+        .unwrap()
```

Comment:
I think this may be something like

```rust
...unwrap()
  .and_then(|label| Label::decode(label.as_ref()))
```

or something like that, up to you.

Reply:
Ah yes, makes sense, thanks!